Bug #36215, replaced 55_db5.1.patch with 55_libgcrypt.patch, 55_db5.1.patch
added libgcrypt-dev and patched libdb5.3-dev to libdb5.1-dev, with 4.2 we
have to use db5.3, but we still need libgcrypt-dev

diff -Nur openldap-2.4.42+dfsg.o/debian/control openldap-2.4.42+dfsg/debian/control
--- openldap-2.4.42+dfsg.o/debian/control       2016-09-02 15:15:27.001998000 +0200
+++ openldap-2.4.42+dfsg/debian/control 2016-09-02 15:15:17.905998000 +0200
@@ -10,6 +10,7 @@ Build-Depends: debhelper (>= 10),
                groff-base,
                heimdal-multidev (>= 7.4.0.dfsg.1-1~) <!stage1>,
                libdb5.3-dev <!stage1>,
+               libgcrypt-dev,
                libgnutls28-dev,
                libltdl-dev <!stage1>,
                libperl-dev (>= 5.8.0) <!stage1>,
